frontend/src/layout/navigation/CloudAnnouncement.tsx

```diff
@@ -1,10 +1,24 @@
 import React from 'react'
-import { Alert } from 'antd'
+import { Alert, Space, Button } from 'antd'
 
 export function CloudAnnouncement({ message }: { message: string }): JSX.Element | null {
+    const parsed_message = message.split('_').join(' ');
+    const github_issue_regex = /ph-([0-9]{1,8})/ig;
```

Comment:
this seems a bit suboptimal - maybe we can do a look-behind and just grab the number directly?

Also from a quick test I ran on this it seems we pick the last and not first issue?

Reply:
Unfortunately Lookbehind's are not supported in [Safari](https://caniuse.com/?search=lookbehind)

I'm not sure if this is a more optimal solution, however:

- I've split it so I use a different regex for the extraction and the replace - so I can ensure that the extraction only captures the first instance.
- I've tested it and it definitely only uses the first one in the link

Reply:
great reference - without the lookbehind the previous one should be fine too